types/react/index.d.ts

```diff
@@ -2236,12 +2236,23 @@ declare namespace React {
 
     interface ScriptHTMLAttributes<T> extends HTMLAttributes<T> {
         async?: boolean;
+        /** @deprecated */
         charSet?: string;
         crossOrigin?: string;
         defer?: boolean;
         integrity?: string;
         noModule?: boolean;
         nonce?: string;
+        referrerpolicy?:
```

Comment:
It's lowercased because React does not recognize `referrerPolicy` prop on a DOM element

Reply:
@eps1lon I've seen your comment in [another PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/48580#discussion_r501087614) about lowercase attributes. Does it mean the `referrerpolicy` attribute should not be added?

Reply:
We probably want to get this into React itself first if they don't already have `referrerPolicy`.

Reply:
Thanks for response. I've removed this change.